### PR TITLE
added try good first issue

### LINKED ISSUE #52: Add a try good first issue section in main website
**Is your feature request related to a problem? Please describe.**
Add a button in the main website which will have link to all the good first issues of this repository , show your creative work you may use svgs to make it cool , more innovative ideas are welcome feel free to suggest design

### PR DESCRIPTION
**Pull request by shreyanshi16**
Fixes #52

#### Changes done

- [Added try good first issue section ]

Screenshots of the changes (If any) -
image

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

#### ✅️ By submitting this PR, I have verified the following

- [ ✅️] Worked on forked repo and starred the main repo
- [ ✅️] Checked to see if a similar PR has already been opened 🤔️
- [ ✅️] Reviewed the contributing guidelines 🔍️
- [ ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
remove yarn lock and change the violet colour change it to blue and point the button to level1 issues as we have no good first issues

**Comment by shreyanshi16** [comment]
@ayan-biswas0412 I have made all the changes. Please tell me if you need any other changes.

**Review by ayan-biswas0412 — DISMISSED** [review]
good work

**Comment by shreyanshi16** [comment]
Thank you!

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
this PR was approved but you have messed up your work

**Comment by shreyanshi16** [comment]
Yeah, I am so sorry. Actually, I was working on the contact-us page and mistakenly I made a commit on this branch. I also tried to uncommit the last commit but that didn't work out. Can you please guide me on how to fix this code again.

**Comment by ayan-biswas0412** [comment]
> Yeah, I am so sorry. Actually, I was working on the contact-us page and mistakenly I made a commit on this branch. I also tried to uncommit the last commit but that didn't work out. Can you please guide me on how to fix this code again.

see this https://www.freecodecamp.org/news/how-to-undo-changes-in-git-e1da7930afdb/

**Comment by shreyanshi16** [comment]
Code fixed! Thank you for your help :)

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
most of the ci checks are failed

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
ci checks are still failing

**Comment by shreyanshi16** [comment]
Yes i am trying to debug them.

**Comment by aniketsingh98571** [comment]
> Yes i am trying to debug them.

@shreyanshi16 I think there is a problem with your `upstream`....

**Comment by shreyanshi16** [comment]
Hey, @aniketsingh98571 I have tried all the possible ways to resolve it but failed. If you have any idea of how to resolve it then will you please guide me?

**Comment by aniketsingh98571** [comment]
@shreyanshi16 
1. You must be missing some "}" or something in your package.json file just check it out or compare it with the old version of your file it helped me out many times
2.  forget to give a comma "," in my JSON file.

**Comment by aniketsingh98571** [comment]
@shreyanshi16  Please refer [this](https://dev.to/redslug/7-solutions-to-ejsonparse-or-unexpected-token-npm-install-error-20nh)

**Comment by shreyanshi16** [comment]
@aniketsingh98571 Thank you for your effort. I tried all solutions you suggested but I am not being able to resolve them.

File: hotelontouch/package.json [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
-}
+    "name": "hotelontouch",
```
Comment (aniketsingh98571) [inline hotelontouch/package.json:2]: Where is opening bracket `{` before `"name":"hotelontouch"`

**Review by aniketsingh98571 — CHANGES_REQUESTED** [review]
Have you changed package.json ...

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
all the ci s are failing

**Pull request closed without merging**